openedx/adg/lms/webinars/helpers.py

```diff
@@ -78,3 +93,86 @@ def send_webinar_registration_email(webinar, email):
         'webinar_link': webinar.meeting_link,
         'webinar_start_time': webinar.start_time.strftime('%b %d, %Y %I:%M %p GMT'),
     })
+
+
+def schedule_webinar_reminders(user_email, webinar):
+    """
+    Schedule reminders for a webinar on mandrill.
+
+    Args:
+        user_email (str): User email to schedule a reminder.
+        webinar (Webinar): The webinar for which reminders will be scheduled.
+    """
+    send_webinar_emails(
+        MandrillClient.WEBINAR_TWO_HOURS_REMINDER,
+        webinar.id,
+        webinar.title,
+        webinar.description,
+        webinar.start_time,
+        [user_email],
+        webinar.start_time - timedelta(hours=2),
```

Comment:
Actually I haven't added this check deliberately. In case of this reminder, email says that "Your webinar is starting soon" and That I think is valid still if time is less than 2 hours, But in case of week before, It is not valid so that is why I have added check for that.